content/numpy/concepts/math-methods/terms/abs/abs.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: '.abs()'
+Description: 'Calculates the absolute value of a number or each element in an array.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'NumPy'
+  - 'Arrays'
+  - 'Functions'
+  - 'Math'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+The **`.abs()`** function calculates the absolute value of a number or the absolute value of all of the elements in an array. The absolute value of a number is the number but without a sign, or the number's distance from zero. `.abs()` can also compute the absolute value of complex numbers in the form of $a + bi$ by computing $\sqrt{a^2 + b^2}$. Additionally, `.absolute()` can be used instead of `.abs()`.
+
+## Syntax
+
+```psuedo
+`numpy.absolute(array, out=None, where=True)`
+```
+
```

Comment:
Also, add the syntax for `numpy.abs()`

Reply:
I changed numpy.absolute to numpy.abs in the latest commit since numpy.abs is the title of the markdown file.